opm/simulators/wells/WellInterfaceGeneric.cpp

```diff
@@ -714,4 +718,74 @@ checkNegativeWellPotentials(std::vector<double>& well_potentials,
     }
 }
 
+void WellInterfaceGeneric::
+updateFiltrationParticleVolume(const double dt, const size_t water_index,
+                               const WellState& well_state, std::vector<double>& filtration_particle_volume) const
+{
+    if (!this->isInjector()) {
+        return;
+    }
+
+    const auto injectorType = this->well_ecl_.injectorType();
+    if (injectorType != InjectorType::WATER) {
+        return;
+    }
+
+    const double conc = this->well_ecl_.getFilterConc();
+    if (conc == 0.) {
+        return;
+    }
+
+    // it is currently used for the filter cake modeling related to formation damage study
+    auto& ws = well_state.well(this->index_of_well_);
+    const auto& connection_rates = ws.perf_data.phase_rates;
+
+    const size_t np = well_state.numPhases();
```

Comment:
Would you mind switching to `std::size_t` from `<cstddef>`, please?

Reply:
will do. Thanks.